ui/public/locales/pt_BR.json

```diff
@@ -1,879 +1,1143 @@
+"label.action.create.template.from.vm": "Criar template a partir da VM",
+"label.action.create.template.from.volume": "Criar template a partir do disco",
+"label.action.create.template.processing": "Criando template....",
 "label.action.create.vm": "Criar VM",
 "label.action.create.vm.processing": "Criando VM....",
-"label.action.create.volume": "Criar Disco",
-"label.action.create.volume.processing": "Criando Disco....",
+"label.action.create.volume": "Criar disco",
+"label.action.create.volume.processing": "Criando disco....",
 "label.action.delete.account": "Remover conta",
 "label.action.delete.account.processing": "Removendo conta....",
-"label.action.delete.cluster": "Remover Cluster",
-"label.action.delete.cluster.processing": "Removendo Cluster....",
-"label.action.delete.disk.offering": "Remover Oferta de Disco",
-"label.action.delete.disk.offering.processing": "Removendo Oferta de Disco....",
-"label.action.delete.domain": "Remover Dom\u00ednio",
-"label.action.delete.domain.processing": "Removendo Dom\u00ednio....",
+"label.action.delete.backup.offering": "Remover uma oferta de backup",
+"label.action.delete.cluster": "Remover cluster",
+"label.action.delete.cluster.processing": "Removendo cluster....",
+"label.action.delete.disk.offering": "Remover oferta de disco",
+"label.action.delete.disk.offering.processing": "Removendo oferta de disco....",
+"label.action.delete.domain": "Remover dom\u00ednio",
+"label.action.delete.domain.processing": "Removendo dom\u00ednio....",
+"label.action.delete.egress.firewall": "Remover regra de sa\u00edda do firewall",
 "label.action.delete.firewall": "Remover regra de firewall",
-"label.action.delete.firewall.processing": "Removendo Firewall....",
-"label.action.delete.ingress.rule": "Remover Regra de Entrada",
-"label.action.delete.ingress.rule.processing": "Removendo Regra de Entrada....",
-"label.action.delete.ip.range": "Remover Range IP",
-"label.action.delete.ip.range.processing": "Removendo Range de IP....",
-"label.action.delete.iso": "Removendo ISO",
+"label.action.delete.firewall.processing": "Removendo firewall....",
+"label.action.delete.ingress.rule": "Remover regra de entrada",
+"label.action.delete.ingress.rule.processing": "Removendo regra de entrada....",
+"label.action.delete.ip.range": "Remover intervalo de IPs",
+"label.action.delete.ip.range.processing": "Removendo intervalo de IPs....",
+"label.action.delete.iso": "Remover ISO",
 "label.action.delete.iso.processing": "Removendo ISO....",
-"label.action.delete.load.balancer": "Remover regra de balanceador de carga",
-"label.action.delete.load.balancer.processing": "Removendo Load Balancer....",
-"label.action.delete.network": "Remover Rede",
-"label.action.delete.network.processing": "Removendo Rede....",
+"label.action.delete.load.balancer": "Remover regra do balanceador de carga",
+"label.action.delete.load.balancer.processing": "Removendo balanceador de carga....",
```

Comment:
Here the correct would not be `"Removendo regra do balanceador de carga...."`?

Reply:
I could not found any usage for this label, but from context, it should be `"Removendo regra do balanceador de carga...."`